### PR TITLE
tooltip animation with native driver

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

**Review by mandrigin — APPROVED** [review]

File: src/status_im/ui/components/list_item/styles.cljs [hunk 1]
```diff
@@ -29,6 +29,6 @@
    :height        size})
 
 (def error
-  {:bottom-value 0
+  {:bottom-value -15
```
Comment (jeluard) [inline src/status_im/ui/components/list_item/styles.cljs]: Can you comment or make those defs?

File: src/status_im/ui/components/tooltip/styles.cljs [hunk 2]
```diff
@@ -23,7 +23,7 @@
    :align-items :center
    :left        0
    :right       0
-   :bottom      bottom-value
+   :transform   [{:translateY bottom-value}]
```
Comment (jeluard) [inline src/status_im/ui/components/tooltip/styles.cljs:27]: For perf?

**Review by jeluard — APPROVED** [review]

File: src/status_im/ui/components/list_item/styles.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/ui/components/list_item/styles.cljs]: @jeluard I'm not sure what to comment here, I tried to make it look exactly the same way is it was before this change, and seems that these magic numbers come from other magic numbers already existing in styles (hundreds of them). So no comments unfortunately.

File: src/status_im/ui/components/tooltip/styles.cljs (hunk 2 above)
Comment (rasom) [reply src/status_im/ui/components/tooltip/styles.cljs:27]: Yes, for perf. `:bottom` can't be animated using native driver, thus animation works in js thread, loads it with unnecessary calculations and also looks laggy at some circumstances.

**Review by cammellos — APPROVED** [review]

File: src/status_im/ui/components/tooltip/styles.cljs (hunk 2 above)
Comment (rasom) [reply src/status_im/ui/components/tooltip/styles.cljs:27]: I changed it a bit so that instead of adding new magic number we only change number's sign :D

**Comment by Serhy** [comment]
No regressions found with tooltips on Android (6.0.1, 8.0, 8.1), iOS 12, MacOS.
Thanks @rasom !

**Pull request merged**